.vscode/scripts/run-seatservice.sh

```diff
@@ -14,57 +14,50 @@
 # SPDX-License-Identifier: Apache-2.0
 
 echo "#######################################################"
-echo "### Running Seatservice                             ###"
+echo "### Running Seat Service                            ###"
 echo "#######################################################"
 
 ROOT_DIRECTORY=$( realpath "$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )/../.." )
 source $ROOT_DIRECTORY/.vscode/scripts/exec-check.sh "$@" $(basename $BASH_SOURCE .sh)
 
-SEATSERVICE_VERSION=$(cat $ROOT_DIRECTORY/prerequisite_settings.json | jq .seatservice.version | tr -d '"')
-SEATSERVICE_PORT='50051'
-SEATSERVICE_GRPC_PORT='52002'
-sudo chown $(whoami) $HOME
+SEATSERVICE_IMAGE=$(cat $ROOT_DIRECTORY/prerequisite_settings.json | jq .seatservice.image | tr -d '"')
+SEATSERVICE_TAG=$(cat $ROOT_DIRECTORY/prerequisite_settings.json | jq .seatservice.version | tr -d '"')
```

Comment:
same, here, should we use app-manifest?

Reply:
See comments to prerequisite_settings.json